master/internal/trial.go

```diff
@@ -647,6 +708,11 @@ func (t *trial) sendNextWorkload(ctx *actor.Context) error {
 }
 
 func (t *trial) processContainerConnected(ctx *actor.Context, msg containerConnected) error {
+	t.lastContainerConnectedTime = time.Now()
+	if len(t.containers) < t.numContainers {
+		actors.NotifyAfter(ctx, allReadyTimeoutPeriod, allReadyTimeout{})
```

Comment:
blocking: you need to set `runId in the message`

Reply:
This magically fixed the unit test I broke